bin/before_install

```diff
@@ -18,3 +18,13 @@ elif [ ! -d "$spec_manageiq" ]; then
   echo "== Cloning manageiq sample app =="
   git clone https://github.com/ManageIQ/manageiq.git --branch master --depth 1 "$spec_manageiq"
 fi
+
+if [ -n "$CI" ]; then
+  # Gemfile.lock.release only applies to non-master branches and PRs to non-master branches
+  if [[ "$GITHUB_REPOSITORY_OWNER" = "ManageIQ" && "$GITHUB_BASE_REF" != "master" && "$GITHUB_REF_NAME" != "master" && "$GITHUB_REF_NAME" != "dependabot/"* ]]; then
+    echo "== Setup Gemfile.lock.release =="
+    cp -f "$spec_manageiq/Gemfile.lock.release" "$gem_root/Gemfile.lock"
+    ruby -e 'File.write("Gemfile.lock", File.read("Gemfile.lock").gsub(/GIT\n  remote: #{Regexp.escape(ENV["GITHUB_SERVER_URL"])}\/#{Regexp.escape(ENV["GITHUB_REPOSITORY"])}\n  revision: .+\n  branch: .+\n/, "PATH\n  remote: .\n"))'
```

Comment:
Unfortunately, Ruby isn't installed by this point in CI :(  But this has to be here before ruby is installed because it also installs the gems.